### ISSUE TITLE
Recognition of the old genus Physematium (Woodsiaceae) [NOT PASSED]

### ISSUE THREAD

**Issue by gonggashan88** [pteridogroup/ppg#73]
### Author(s) of proposal

Li-Bing Zhang, Ngan Thi Lu

### Name of taxon

Physematium

### Rank of taxon

Genus

### Approximate number of species affected

28

### Description of change

Reinstatement of Physematium (1829) in Woodsiaceae to reflect the deep divergence in the phylogeny, morphological diagnosability, and geographical differentiation.

### Reason for change

Woodsiaceae contain ca. 65 (instead of 30 as early estimated) species that are widely distributed in the north temperate zone and in higher elevations in the tropics, e.g., Central and South America, Africa (Angola, South Africa), and Madagascar.

Two contemporary classifications of Woodsiaceae recognized dramatically different numbers of genera. Shmakov (2015) recognized seven genera: Cheilanthopsis, Eriosoriopsis, Hymenocystis, Physematium, Protowoodsia, Woodsia, and Woodsiopsis. Recognition of Hymenocystis and Protowoodsia would make Cheilanthopsis paraphyletic. Shao et al. (2015) advocated for a single-genus classification with three subgenera: W. subg. Woodsia, W. subg. Cheilanthopsis, and W. subg. Physematium. Their W. subg. Physematium, including species of both Physematium and Woodsiopsis, is paraphyletic based on Lu et al. (2019) tree. Shao et al. (2015) did not sample any taxa of the old name, Hymenocystis. PPG I (2016) followed Shao et al.’s (2015) classification. 

Based on a new phylogeny (tree below) using a largely expanded taxon sampling (122 samples representing ca. 43 species vs. 33 samples representing. 21 spp. in Shao et al. (2015)), detailed analyses of character evolution (figure below), ancient deep divergence (ca. 45 million years ago; Larsson, 2014), and geographical distribution, Lu et al. (2019) advocated a two-genus classification: Woodsia (1810) and Physematium (1829), corresponding to the two subfamilies proposed by Shmakov (2015). Geographically, ca. 37 species of Woodsia are distributed in circumboreal areas and East Asia only, while ca. 28 species of Physematium occur in the Americas, Asia, and Africa and Madagascar. Also, species of Woodsia have x = 39 or 41, while those of Physematium have x = 33, 37, 38, or 41.

![Image_20240212223518](https://github.com/pteridogroup/ppg/assets/126826777/4c525593-0529-4401-9bc6-8e8bf1ebd442)
![Figure_8_selected A-M copy](https://github.com/pteridogroup/ppg/assets/126826777/ff8f6d7f-6572-4db9-bfe7-5c1eff0ef1fb)

### Reference(s) for publication of the name

[Shmakov 2015_Woodsiaceae.pdf](https://github.com/pteridogroup/ppg/files/14269469/Shmakov.2015_Woodsiaceae.pdf)
[Shao et al 2015_Woodsiaceae.pdf](https://github.com/pteridogroup/ppg/files/14269479/Shao.et.al.2015_Woodsiaceae.pdf)
[Lu et al. 2019_Woodsia.pdf](https://github.com/pteridogroup/ppg/files/14269481/Lu.et.al.2019_Woodsia.pdf)

### Code of Conduct

- [X] I agree to follow the PPG Code of Conduct

**Comment by leonperrie** [comment]
Hello Li-Bing,

PPG I said it sought “to preserve existing taxa and circumscriptions that are widely accepted and consistent with our understanding of lycophyte and fern phylogeny” (p. 565).  

PPG I recognised a broadly circumscribed _Woodsia_ (with one genus in the family).  Your work and that of others shows that _Woodsia_ sensu PPG I is monophyletic.

I believe my experience that general users of taxonomic names dislike name changes is not unusual.  Many taxonomists recognise the desirability of trying to meet the wants of general users for stability whilst constructing a taxonomic classification based on monophyletic groups (e.g., Drew et al., 2017, Taxon 66: 133-145).  This might equate to ‘be monophyletic, and make the fewest name changes to get there’.

As far as I know, PPG does not have standardised criteria for how old a genus or family should be.

Your work and that of others has greatly extended the understanding of relationships within Woodsiaceae.  But that could alternatively be represented taxonomically using infra-generic ranks, which would have the advantage of general users not having to incur name changes.

In summary, _Woodsia_ sensu PPG I is consistent with phylogeny and if PPG wants “to preserve existing taxa and circumscriptions” as much as possible, why make a change?

More generally, if we split a monophyletic group into multiple monophyletic sub-groups (or lump multiple monophyletic groups), then when does taxonomic revision ever end?

**Comment by gonggashan88** [comment]
Hi Leon, thanks for your comments. You made good point! Yes, you can use subgenera or sections for such monophyletic clades with the family. On the other hand, those infrageneric ranks are almost always ignored except by professional plant taxonomists or even only by specialists of a particular plant group. 

Placing all the members of the family in one genus is convenient, but adopting two-genus classification informs science (phylogeny) better and emphasizes closer relationships within each of _Physematium_ and _Woodsia_ s.n. or more distant relationships between species in two genera. These two genera are deeply diverged (ca. 45 million years ago) and morphologically diagnosable anyway. Also, _Physematium_ is an old name.

I am sure when _Polypodium_ s.l. or _Lycopodium_ s.l. was split decades ago, there were botanists who were against the splitting using the similar argument you mentioned (even now Kew's Plants of the World Online recognizes one genus only in Grammitidoideae). Think how many genera within each of them PPG I recognized! Similarly, future generations might find the two-genus classification of Woodsiaceae more informative and useful.  Convenience alone is not enough. A better science-informing classification will survive in the end.

Best wishes, Libing
​Li-Bing Zhang, Dr. nat., Curator; Missouri Botanical Garden 
[www.mobot.org/Li-Bing Zhang](https://www.missouribotanicalgarden.org/plant-science/plant-science/research-staff/article/2628/zhang-li-bing)

**Comment by leonperrie** [comment]
Thanks Li-Bing.  If only there was a way to objectively determine the most informative (for science or otherwise) classification among alternative monophyletic-based options.  But there doesn’t seem to be.  Even morphological diagnosability is open to interpretation.

Which seems to leave the choice as subjective, which is often convenience in some form or other.

In which case, should convenience for a few taxonomists outweigh the convenience of the many more general users?  I’m mindful that it’s the latter, through their taxes, paying the salaries of those of us fortunate to be professional taxonomists.

**Comment by gonggashan88** [comment]
Thanks, Leon! A new classification is inconvenient for everybody who is familiar with the existing one. When inconvenience/laziness is not the case for future pteridologists (beyond the PPG II classification) or outside of the community of pteridology, will they choose a better science-informing classification or a conservative and convenient/lazy classification? 

In that sense, the more aligned the PPG II classification is with science, the better the chance it will be appreciated.

**Comment by joelnitta** [comment]
This proposal was voted on during PPG Ballot 10 (voting period April 2024). A total of 65 votes were cast. There were 30 'Yes' votes (46.2%) and 35 'No' votes (53.8%). The proposal does not pass.
